addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -86,7 +86,14 @@ const GallerySliderWidget = publicWidget.Widget.extend({
         this.$prev = this.$indicator.find('li.o_indicators_left').css('visibility', ''); // force visibility as some databases have it hidden
         this.$next = this.$indicator.find('li.o_indicators_right').css('visibility', '');
         var $lis = this.$indicator.find('li[data-slide-to]');
-        var nbPerPage = Math.floor(this.$indicator.width() / $lis.first().outerWidth(true)) - 3; // - navigator - 1 to leave some space
+        var indicator_width = this.$indicator.width();
+        if (!indicator_width){
+            var clone = this.$indicator.clone();
```

Comment:
For new code, use ES6.
Here, `const` instead of `var`. Also as the variable contains a jquery element, it should start with `$`